src/sidebar/media-embedder.js

```diff
@@ -11,16 +11,47 @@ function audioElement(src) {
   return html5audio;
 }
 
+/**
+ * Wrap an element in a container that causes the element to be displayed at
```

Comment:
I've used this hack before on a personal project! This function is well-commented, thank you.